_includes/badges_load_dynamic_roles_recognition.html

```diff
@@ -0,0 +1,24 @@
+{% assign authors = site.categories.blog | map: 'username' %}
```

Comment:
@loleg yeah. The purpose of this file is to load the role and recognitions that Jekyll already knows about (who is an author, who is a project maintainer) into the common user_roles attribute to avoid having stale or out-of-date info. The info is already in Jekyll's brain, so why not spit it out into the user_roles attribute?

However, I also see the benefit of pulling back from excessive logic and just writing the attributes directly. After all, issuing badges seems like it should be an "intentional" thing done by a human. Not to mention that this imposes a tax on Jekyll's build time.

So I'm happy to remove this file entirely and work on another solution for issuing these kinds of "automatic" badges correctly.